### ISSUE TITLE
Relationship between `Fedora`, `CentOS`, `RHEL` and `AlmaLinux`

### ISSUE THREAD

**Issue by srbala** [AlmaLinux/wiki#79]
Convert @carlwgeorge tweeter post into a wiki page

https://chat.almalinux.org/almalinux/pl/imsbb6w5kfgbfm6cw6qskrpbqa

https://twitter.com/carlwgeorge/status/1439724277746573314

**Comment by bennyvasquez** [comment]
I think this would be an awesome thing to add to the top of this page: https://wiki.almalinux.org/Comparison.html In my mind you can omit how things used to work because that'll just add more confusion.  

Just speak to how it works now. I bet Carl would even be okay with you using (as long as you cite the source) those images.

**Comment by GrooveCS** [comment]
Hi again @bennyvasquez & @alaurie 

I was reviewing this issue and the marketing chat, and I saw that @alaurie had created a diagram that would be useful for this issue. 

https://chat.almalinux.org/almalinux/pl/d59n39gniprmjgxqs413any3ca

**Comment by alaurie** [comment]
@GrooveCS @bennyvasquez I don't think we reached a general consensus on how to format the image, how Kitten runs alongside the point releases. Just need to confirm that and can then add it.
